app.py

```diff
@@ -23,7 +23,7 @@ def init_app(self):
         tk_multi_breakdown = self.import_module("tk_multi_breakdown")
         cb = lambda: tk_multi_breakdown.show_dialog(self)
         self.engine.register_command(
-            "Scene Breakdown...", cb, {"short_name": "breakdown"}
+            "{}...".format(self.get_setting("display_name")), cb, {"short_name": "legacy_breakdown"}
```

Comment:
hey @rob-aitchison , not sure about changing the `short_name` here...
That could break the workflow of some users who are calling the app in a script and use it in "batch" mode
Also, you don't have a "display_name" setting in your app configuration so `get_setting("display_name")` will return None. If you want to use [this setting](https://github.com/shotgunsoftware/tk-multi-breakdown/blob/SG-30515/info.yml#L38), you need to call `self.display_name` instead. But be careful that this value cannot be changed in the config...